Sources/NetworkExtensions/REST/RESTClient.swift

```diff
@@ -45,7 +47,7 @@ public protocol RESTClient {
     var requiredQueryParameters: [QueryParameter] { get }
 
     /// URL Session to be used for all requests.
-    var session: URLSessionProtocol { get }
+    var session: Session { get }
```

Comment:
Will need that because eventually we can make this not only URLSessionProtocol but also LongPollingProtocol. So I can add extensions for when Session: LongPollingProtocol with requests for that end.